src/javascripts/ng-admin.js

```diff
@@ -11,9 +11,12 @@ require.config({
         'angular-resource': 'bower_components/angular-resource/angular-resource',
         'angular-sanitize': 'bower_components/angular-sanitize/angular-sanitize',
         'angular-ui-router': 'bower_components/angular-ui-router/release/angular-ui-router',
+        'bootstrap': 'bower_components/bootstrap/dist/js/bootstrap',
```

Comment:
seems that only bootstrap transitions and collapse are needed at this point (http://eonasdan.github.io/bootstrap-datetimepicker/Installing/).